docs/sources/administration/configuration.md

```diff
@@ -450,203 +531,293 @@ Text used as placeholder text on login page for login/username input.
 
 Text used as placeholder text on login page for password input.
 
-<hr>
+### default_theme
 
-## [auth]
+Set the default UI theme: `dark` or `light`. Default is `dark`.
 
-Grafana provides many ways to authenticate users. The docs for authentication has been split in to many different pages
-below.
+### External user management
 
-- [Authentication Overview]({{< relref "../auth/overview.md" >}}) (anonymous access options, hide login and more)
-- [Google OAuth]({{< relref "../auth/google.md" >}}) (auth.google)
-- [GitHub OAuth]({{< relref "../auth/github.md" >}}) (auth.github)
-- [Gitlab OAuth]({{< relref "../auth/gitlab.md" >}}) (auth.gitlab)
-- [Generic OAuth]({{< relref "../auth/generic-oauth.md" >}}) (auth.generic_oauth, okta2, auth0, bitbucket, azure)
-- [Basic Authentication]({{< relref "../auth/overview.md" >}}) (auth.basic)
-- [LDAP Authentication]({{< relref "../auth/ldap.md" >}}) (auth.ldap)
-- [Auth Proxy]({{< relref "../auth/auth-proxy.md" >}}) (auth.proxy)
+external_manage_link_url =
+external_manage_link_name =
+external_manage_info =
 
-### login_cookie_name
+### viewers_can_edit
 
-The cookie name for storing the auth token, the default is `grafana_session`.
+Viewers can edit/inspect dashboard settings in the browser, but not save the dashboard.
+Default is `false`.
 
-### oauth_state_cookie_max_age
+### editors_can_admin
 
-How long the OAuth state cookie lives before being deleted. Default is `60` (seconds)
-Administrators can increase this if they experience OAuth login state mismatch errors.
+Editors can administrate dashboards, folders and teams they create.
+Default is `false`.
 
-## [dataproxy]
+<hr>
 
-### logging
+## [auth]
 
-This enables data proxy logging, default is `false`.
+Grafana provides many ways to authenticate users. Refer to the Grafana [Authentication overview]({{< relref "../auth/overview.md" >}}) and other authentication documentation for detailed instructions on how to set up and configure authentication.
 
-### timeout
+### login_cookie_name
 
-How long the data proxy should wait before timing out. Default is `30` (seconds)
+The cookie name for storing the auth token. Default is `grafana_session`.
 
-### send_user_header
+### login_maximum_inactive_lifetime_days
 
-If enabled and user is not anonymous, data proxy will add X-Grafana-User header with username into the request. Default is `false`.
+The lifetime (days) an authenticated user can be inactive before being required to log in at next visit. Default is 7 days.
 
-<hr />
+### login_maximum_lifetime_days
```

Comment:
Are these duplicated on multiple pages now? Is this really necessary when you link to the Authentication overview above?

Reply:
We don't actually document these in the Authentication overview topic. The only mention of most of these options is an example with no explanation.

Reply:
Good